source/getting_started/client_configuration.rst

```diff
@@ -0,0 +1,171 @@
+        is an advanced setting for developers who compile their own Amalgam
+        shared object binaries.
+
+        - ``arch``
+
+          Target CPU architecture of the amalgam library. This is determined
+          programmatically by default and should only be overridden here by
+          advanced users.
+
+          Valid values are:
+
+          - ``x86_64`` - Most Windows, older MacOS, and linux machines
+          - ``arm64`` - Modern MacOS and some Linux machines
+          - ``arm64_8a`` - Specifically compiled for CPUs compatible with
+            generation 1 ``aarch64`` instruction sets such as early Raspberry
+            Pi's and some early Graviton processors.
+
+      - ``execution_trace_dir``
+
+        Set the desired path where tracefiles should be written. The path
+        must be a valid directory with write permission for the
+        operating user. The default value is the current working directory.
+
+      - ``execution_trace_file``
+
+        Set the desired file name for the trace files. Default value is
+        ``<trainee_name>.trace``.
+
+      - ``library_postfix``
+
+        The Amalgam shared libraries are compiled for different modes of
+        operation. Each of the valid values for this option are:
+
+        - ``-mt`` for Multithreaded operation. This is almost always the best
+          option (and is thus the default on most systems).
+        - ``-st`` for Singlethreaded operation. In rare cases usually involving
+          small Trainee models, this may be the best option.
+        - ``-openmp`` for Multi Processing using the `OpenMP <https://openmp.org>`_ library.
+          This option may be useful when latency is paramount.
+        - ``-pgc`` for "Pedantic Garbage Collection". This is exclusively used
+          by developers in Amalgam debugging scenarios.
+
+        - ``os``
+
+          Target operating system of the amalgam library. Valid values are:
+
+          - ``linux``
+          - ``darwin``
+          - ``windows``
+
+      - ``trace``
+
+        - ``false`` (default) - No trace files are written.
+        - ``true`` - Use this option to generate tracefiles.
+
+    - ``core``
+
+      This section is for changing the behavior of the Howso Engine (the Amalgam
+      shared objects).
+
+      - ``persisted_trainees_dir``
+
+        By default, persisted trainees are stored in the current working directory.
+        Set this option to a valid path with sufficient write permissions to
+        store all persisted trainees if so desired.
+
+    - ``howso_path``
+
+      Specify a specific location to look for the Howso Engine (core). This is
+      an advanced setting used by developers.
+
+    - ``howso_fname``
+
+      Specify a specific filename for the Howso Engine (core). This is an
+      advanced setting used by developers.
+
+    - ``trainee_template_path``
+
+      Specify the path to look for the howso-template trainee_template_fname -
```

Comment:
Overuse of Specify, mix it up a little. Also, the "mood" feels off as suggested in above comments.